packages/embeds/embed-core/src/EmbedElement.ts

```diff
@@ -171,6 +177,10 @@ export class EmbedElement extends HTMLElement {
       return;
     }
     this.layout = newLayout;
+    // We can't accidentaly show skeleton plus no need to change it if it isn't showing
+    if (!this.isSkeletonLoaderVisible()) {
```

Comment:
It was causing a bug, in production too, where when we resize the screen, the skeleton shows up